x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -878,4 +880,42 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
       }
     })
   );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/bulk_create Bulk create jobs and datafeeds
+   * @apiName BulkCreateJobs
```

Comment:
This will need adding to `apidoc.json`

Reply:
https://github.com/elastic/kibana/pull/101037/commits/04c01e9a1778555b0c7480ed2ec46c1f5567a2da